tools/generators/migrate-converged-pkg/index.ts

```diff
@@ -58,6 +72,8 @@ export default async function (tree: Tree, schema: MigrateConvergedPkgGeneratorS
   updateNpmScripts(tree, options);
   updateApiExtractorForLocalBuilds(tree, options);
 
+  updateNxWorkspace(tree, options);
```

Comment:
What about the packages that are currently already migrated ? should those tags be added in this PR ?

Reply:
you can run it again on those :)